### DISCUSSION TITLE
[Esame 23 Giugno 2020] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#239] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8b071a7f-fe9c-4cf0-91ff-e127c78ee26c)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/a5cf6291-cbc5-4546-a34a-3174f1f9f5fe)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/2c611e60-6eff-429e-b514-1769a7925dce)

**Comment by Elia-Belli** [comment]
**(1)** S conosce la prima domanda con probabilità $60\\%$ e in tal caso risponderà sicuramente in modo corretto, altrimenti risponderà correttamente con probabilità $1/4$:
$$\mathbb{P}(1° corretta)=\mathbb{P}(sa\ la\ risposta)\cdot 1+\mathbb{P}(non\ sa\ la\ risposta)\cdot \frac{1}{4}=0.6+0.4\cdot \frac{1}{4}=0.7$$

**(2)** Sia $C$ la v.a. che indica il numero di risposte corrette, siccome la risposta di ogni domanda è indipendente dalle altre e la probabilità di rispondere correttamente alla singola domanda è $0.7$, allora $C\sim Bin(10,0.7)$:
$$\mathbb{P}(C\geq 6)=\sum\_{k=6}^{10}\binom{10}{k}\cdot 0.7^{k}\cdot 0.3^{10-k}$$

**(3)** Utilizziamo la formula della probabilità condizionata:

$$\mathbb{P}(ne\ conosce\ 6|C=6)=\frac{\mathbb{P}(ne\ conosce\ 6 \cap C=6)}{\mathbb{P}(C=6)}=\frac{\binom{10}{6}\cdot 0.6^{6}\cdot 0.3^{4}}{\binom{10}{6}\cdot 0.7^{6}\cdot 0.3^{4}}=(\frac{0.6}{0.7})^6\simeq 0.4$$

Per calcolare $\mathbb{P}(ne\ conosce\ 6 \cap C=6)$ ho considerato che le domande possono essere risposte correttamente solo se le si conoscono, ovvero con $p=0.6$, mentre vengono sempre risposte erroneamente se non le si conoscono con probabilità $0.3$.

In questo caso forse è preferibile considerare le risposte con una v.a. multinomiale, dato che ci interessa la distinzione tra risposte corrette risposte a caso e non. Indichiamo con $C_c$ la v.a. che indica il numero di risposte corrette conosciute, $C_n$ la v.a che indica il numero di risposte corrette a caso e $E$ la v.a che indica il numero di risposte errate:

$$(C_c,C_n,E)\sim Multinom(10,3,0.6,0.1,0.3)$$

Vogliamo quindi calcolare:

$$\mathbb{P}(C_c=6| E=4)=\frac{\mathbb{P}(C_c=6,E=4)}{\mathbb{P}(E=4)}=\frac{\frac{10!}{6!\cdot 0!\cdot 4!}\cdot 0.6^6\cdot 0.1^0\cdot 0.3^4}{\binom{10}{4}0.3^4\cdot 0.7^6}=(\frac{0.6}{0.7})^6\simeq 0.4$$

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli, ma con punto 3 fatto con una semplice binomiale
![immagine](https://github.com/user-attachments/assets/016dce7e-c445-40d4-81cf-8b3cd4f0a4f8)
